### PR TITLE
fix: Ripple effect on the set of commands in ChatActivity

### LINKED ISSUE #2158: Add ripple effect when user clicks on the set of commands .
**Describe the feature you'd like**
The user is not prompted if the command is clicked or not. By mistake, I clicked the command twice or three not knowing if the command is clicked.

**Additional context**
Adding ripple effect on the commands will prompt the user that the command has been clicked

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes #2158 

Changes: Added ripple effect when the user clicks on the commands.
Ripple effect helps us ensure that the command is clicked. Otherwise, the user is in the impression that the command has not been clicked and would click a number of times.

Screenshots for the change: 
ezgif com-video-to-gif (2)

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
@iamareebjamal @batbrain7 @arundhati24 please review !!

**Comment by Bhavnaharitsa** [comment]
@arundhati24 Please review mam!

**Comment by Bhavnaharitsa** [comment]
@iamareebjamal @batbrain7 @arundhati24 please review!!

File: app/src/main/res/drawable/ripple_effect.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (iamareebjamal) [inline app/src/main/res/drawable/ripple_effect.xml]: Remove

File: app/src/main/res/drawable-v21/ripple_effect.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (iamareebjamal) [inline app/src/main/res/drawable-v21/ripple_effect.xml]: Remove

**Pull request closed without merging**